.cirrus.yml

```
@@ -184,7 +184,7 @@ task:
     - netsh int ipv4 set dynamicport tcp start=1025 num=64511
     - netsh int ipv6 set dynamicport tcp start=1025 num=64511
     # Exclude feature_dbcrash for now due to timeout
-    - python test\functional\test_runner.py --nocleanup --ci --quiet --combinedlogslen=4000 --jobs=4 --timeout-factor=8 --extended --exclude feature_dbcrash
+    - python test\functional\test_runner.py --nocleanup --ci --quiet --combinedlogslen=4000 --jobs=6 --timeout-factor=8 --extended --exclude feature_dbcrash
```

```suggestion
    - python test\functional\test_runner.py --nocleanup --ci --quiet --combinedlogslen=4000 --jobs=12 --timeout-factor=8 --extended --exclude feature_dbcrash
```

There should be no risk in saturating the CPU, given enough memory

---

Implemented in https://github.com/bitcoin/bitcoin/pull/26297#issuecomment-1275931299.